docs/adr/lts-strategy.md

```diff
@@ -0,0 +1,176 @@
+   of the project
+3. Middleware: Libraries for use *with* `express` but not installed with it directly
+4. Others: Tools, documentation, websites, resources, etc.
+
+While much of this proposal is targeted specifically at `express`, this terminology will be used to differentiate where
+we are not only talking about `express`.
+
+## Decision
+
+The Express project will have a well defined Long Term Support (LTS) strategy and schedule based on the following
+requirements:
+
+- `express` major versions will go through three supported phases: Current, Active, & Maintenance
+  - `CURRENT`: A new major version is designated as `CURRENT` upon release. It is available but not the `latest` version
+    on npm for a minimum of 3 months.
+  - `ACTIVE`: After the minimum 3 month period and the TC agrees it is stable and secure, the `ACTIVE` version is
+    tagged `latest` on npm for a minimum of 12 months.
+  - `MAINTENANCE`: When a new major version becomes `ACTIVE`, the previous major version enters `MAINTENANCE` for 12 months.
+- After the `MAINTENANCE` time has ended the major version is considered `EOL` and is unsupported.
+- During the 12 month `ACTIVE` period, a new major version may be released but *not* until at least 6 months have passed
+  since the `CURRENT` version became `ACTIVE`.
+- Users are required to follow the head/latest of each major release line for support with all packages (`express`,
+  dependencies, middleware, & tools/other)
+- If we have no necessary breaking changes, we will not release a new major version. Applies to `express`,
+  dependencies, middleware, & tools/other.
+- Dropping old Node.js versions *alone* is not enough of a breaking change to release a new major version. Applies to
+  `express`, dependencies, middleware, & tools/other.
+- `express` version 4.x is considered a special case. It will receive a longer `MAINTENENCE` phase of 18 months or until
+  the TC agrees we are save cutting support.
+- `express` dependency packages will follow the same timeline and support dates for the `express` version which they are
+  included with.
+- Middleware packages should follow the timeline and support dates for the `express` versions they are compatible, but
+  should make reasonable and informed decisions which can deviate if necessary.
+- All other packages are suggested to define an appropriate strategy for support.
+
+### Node.js Version Support
+
+- Each major version of all packages (`express`, deps, middleware, etc) will support a minimum of the oldest Node.js LTS
+at the time of release and maintain that support through its entire life cycle. This includes EOL Node.js versions.
+- `express` may choose to continue support for EOL Node.js versions in all three phases if deemed valuable, but will
+strive to cut support before it becomes a drain on progress.
+- All `express` dependency packages must support, at a minimum, the Node.js versions that are supported by the `express`
```

Comment:
The combination of this and preceding line might mean that when an unsupported version of node.js has a particular vulnerability, dependency packages are forced to provide workarounds to avoid exposing the vulnerability.

Reply:
Hm, interesting, yeah I think I follow but I also think that we have discussed something similar to this as well before. Our discussion was around v4 users stuck on EOL node versions and how our pass through apis expose many of the same vulns as the underling runtime. IIRC we decided this was out of scope for what we consider this projects problem to solve. 

Now that you are saying that though, I wonder if we should add this to our threat model doc? https://github.com/expressjs/security-wg/blob/main/docs/ThreatModel.md cc @expressjs/security-wg

Reply:
I think it's ok to go the other way and explicitly say that express and it's ecosystem will not be forced to work around vulns in EOL Node.

Reply:
We briefly discussed today, and we have agreed to remove some of this language from this doc, to keep it focused specifically on release strategy and to remove excess details that do not need to be explicitly stated or promised.

Reply:
I removed this section and called out our plan to address this independently.